src/env-inl.h

```diff
@@ -206,6 +206,24 @@ inline AsyncHooks::DefaultTriggerAsyncIdScope ::~DefaultTriggerAsyncIdScope() {
     old_default_trigger_async_id_;
 }
 
+inline void AsyncHooks::push_execution_async_resource(
+  v8::Local<v8::Value> execution_async_resource) {
+  execution_async_resources_.push(v8::Global<v8::Value>(
+    env()->isolate(), execution_async_resource));
```

Comment:
Suggestion:

```c++
  execution_async_resources_.emplace(
      env()->isolate(), execution_async_resource);
```

(This avoids creating a temporary `v8::Global`)

Reply:
Good idea. Much cleaner looking too. :tada: